cartofacade/capi.go

```diff
@@ -118,7 +118,6 @@ type CartoAlgoConfig struct {
 	MaxRange             float32
 	MinRange             float32
 	UseIMUData           bool
-	UseOdometerData      bool
```

Comment:
Removed this because there is no such configuration parameter in Cartographer. In order to use odometry data all we have to do is to add the sensor data to the map builder - no flag has to be enabled.